ur_driver/src/ur_driver/driver.py

```diff
@@ -188,7 +188,12 @@ def __trigger_halted(self):
         log("Halted")
 
     def __on_packet(self, buf):
-        state = RobotState.unpack(buf)
+        try:
+            state = RobotState.unpack(buf)
+        except Exception as e:
+            rospy.logerr("Exception while unpacking RobotState")
```

Comment:
Technically this is incorrect (at least in the case of #183): the pkt received there is not a `ROBOT_STATE`, but a `ROBOT_MESSAGE`. Properly dealing with that would require more extensive changes though.